tests/events/tests_events_register_api.py

```diff
@@ -0,0 +1,109 @@
+from rest_framework.test import APITestCase, APIClient
+from rest_framework import status
+from events.models import Event
+from users.models import CustomUser
+from tests.AuthAPITestCase import AuthAPITestCase
+
+class EventRegisterTestCase(AuthAPITestCase):
+    def setUp(self):
+        super(EventRegisterTestCase,self).setUp()
+        """
+        Create different events with different years and flag input in test database
+        """
+        # create an unverifeid authenticated user
+        self.auth_unverified_user_email = "crash.test1.dummy@gmail.com"
+        self.auth_unverified_user_password = "test1.modelx"
+        self.auth_unverified_user = CustomUser.objects.create_user(
+            email = self.auth_unverified_user_email,
+            username = self.auth_unverified_user_email,
+            password = self.auth_unverified_user_password,
+            verified=False
+        )
+        
+        # create an verified authenticated user
+        self.auth_verified_user_email = "crash.test2.dummy@gmail.com"
+        self.auth_verified_user_password = "test2.modelx"
+        self.auth_verified_user = CustomUser.objects.create_user(
+            email = self.auth_verified_user_email,
+            username = self.auth_verified_user_email,
+            password = self.auth_verified_user_password,
+            verified=True
+        )
+        
+        login_api="/users/login/"
+        client = APIClient()
+        # get auth token of verified user
+        verified_login_payload = {
+            'email':self.auth_verified_user_email,
+            'password':self.auth_verified_user_password
+        }
+        verified_login_response = client.post(login_api,verified_login_payload)
+        self.verified_auth_token = verified_login_response.data['token']
```

Comment:
Since you have already imported the `AuthAPITestCase` class, you can now make use of its methods like `create_auth_client` instead of writing all of this code over and over again ✌️